modules/openapi-generator/src/main/resources/cpp-rest-sdk-client/model-source.mustache

```diff
@@ -7,6 +7,90 @@
 namespace {{this}} {
 {{/modelNamespaceDeclarations}}
 
+
+{{#isEnum}}
+
+{{classname}}::{{classname}}()
+{
+}
+
+{{classname}}::~{{classname}}()
+{
+}
+
+void {{classname}}::validate()
+{
+    // TODO: implement validation
+}
+
+web::json::value {{classname}}::toJson() const
+{
+    web::json::value val = web::json::value::object();
+
+    {{#allowableValues}}{{#values}}
+    if (m_value == e{{classname}}::{{classname}}_{{.}}) val = web::json::value::string(U("{{.}}"));{{/values}}{{/allowableValues}}
+
+    return val;
+}
+
+void {{classname}}::fromJson(const web::json::value& val)
+{
+    auto s = val.as_string();
+
+    {{#allowableValues}}{{#values}}
+    if (s == L"{{.}}") m_value = e{{classname}}::{{classname}}_{{.}};{{/values}}{{/allowableValues}}
```

Comment:
Instead of wchar `L` which works on VC++, could this be `utility::string_t("{{.}}")` instead, would this work both on VC++ and g++?

Reply:
Sorry about that... Should have noticed this pattern everywhere. My branch now uses utility::conversions::to_string_t("{{.}}")